src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,40 @@
 package mate.academy;
 
+import mate.academy.dao.BookDao;
+import mate.academy.lib.Injector;
+import mate.academy.model.Book;
+import java.math.BigDecimal;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.academy.dao");
```

Comment:
Ensure that the package name 'mate.academy.dao' used in `Injector.getInstance` matches the actual package structure of your project. If the package name is incorrect, it might lead to runtime errors when trying to get an instance of `BookDao`.